### DISCUSSION TITLE
Forcing requests to be made by the host not the client

### DISCUSSION THREAD

**Discussion by BrunoGelli** [gapitio/gapit-htmlgraphics-panel#183] [Q&A]
Hi fellow grafaners,

I have a server running Grafana, it does have access to several vlans for different data sources. Some of those are private, e.g. a security camera vlans. It is imperative this vlans do not have public access.

I'm creating a panel using HTMLGraphics that would display live feed from this cameras. Everything works fine if the client (laptop, phone or anything else than the server) is on the private vlan, but if the client is somewhere else it does not work. 

I narrowed the bug down to understanding that the requests are made on the client-side. Can I force the requests to be made by the host? That is, when I add a HTMLGraphic panel, the http request is done by the server itself.

Thank you so much,
Bruno

**Comment by ZuperZee** [comment]
Maybe this guide is helpful https://gapit-htmlgraphics-panel.gapit.io/docs/guides/send-post-requests-through-grafana/
